doc/sphinx/how-to/how-to-control-decorations.md

```diff
@@ -0,0 +1,40 @@
+# How to Control Decorations
+A big part of writing your own compositor is to be able to customize its looks,
+behaviour, and effects to your own liking. One way of customizing and unifying
+the looks of applications is server side decorations. MirAL allows you to
+customize how the server deals with decoration requests using
+{class}`miral::Decorations`.
```

Comment:
This text will confuse. Users may reasonably expect to customise the decorations (colour, position of buttons, style, ...) 

We need to make it clear that this HOWTO is about choosing CSD or SSD, it isn't about customising decorations.